holoviews/tests/core/data/testibisinterface.py

```diff
@@ -0,0 +1,290 @@
+import sqlite3
+from unittest import SkipTest
+
+from tempfile import NamedTemporaryFile
+
+try:
```

Comment:
on ibis master branch, now all the backends are inside ibis.backends https://github.com/ibis-project/ibis/tree/master/ibis/backends. probably in the next few days the new ibis release will be available.